src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -3058,6 +3084,12 @@ private void AddSynthesizedRecordMembersIfNecessary(MembersAndInitializersBuilde
 
                 var ctor = addCtor(builder.RecordDeclarationWithParameters);
 
+                builder.UpdateIsNullableEnabledForConstructorsAndFields(ctor.IsStatic, compilation, paramList);
+                if (builder.RecordDeclarationWithParameters?.BaseList is { } baseList)
+                {
+                    builder.UpdateIsNullableEnabledForConstructorsAndFields(ctor.IsStatic, compilation, baseList);
```

Comment:
>baseList [](start = 104, length = 8)

I think we should use span of either ```builder.RecordDeclarationWithParameters.PrimaryConstructorBaseType?.ArgumentList``` or ```builder.RecordDeclarationWithParameters.PrimaryConstructorBaseType```, if any, instead. Including entire base list, especially when there are no arguments feel inappropriate.